masterdata/configuration/liquibase/liquibase.xml

```diff
@@ -458,4 +458,55 @@
         </sql>
     </changeSet>
 
+    <changeSet id="GOK-193-inv-Department"  author="Inventory">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">select count(*) from inv_department where name = 'Pharmacy'</sqlCheck>
+        </preConditions>
+        <comment>Creating a Pharmacy Department in the inventory Module</comment>
+        <sql>
+            SET @puuid = uuid();
+            INSERT INTO inv_department (name, description, creator, date_created, uuid)
+            VALUES ('Pharmacy', 'Department where the entire stock is stored/distributed', 1, NOW(), @puuid);
+        </sql>
+    </changeSet>
+
+    <changeSet id="GOK-193-inv-StockRooms-2375843"  author="Inventory">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="6">select count(*) from location where name = 'Hosakote UPHC' OR name = 'Hosakote Nama Clinic' OR name = 'Nandagudi SC1' OR name = 'Nandagudi SC2' OR name = 'Anupahalli SC1' OR name = 'Bengaluru Rural District Hospital'</sqlCheck>
```

Comment:
this condition will fail even if one location is not present in location table. is there any other way to do this?

Reply:
Yes Gokul ideally this approach we can not do for actual implementation when we do for all HWc's across karnataka.Considering pilot as discussed we are considering 6 locations to be present for us to tag them to respective stockrooms.